docs/collection_migration.md

```diff
@@ -7,6 +7,7 @@ Here are some links to read through and familiarize yourself with ...
 * https://www.ansible.com/blog/getting-started-with-ansible-collections
```

Comment:
Could we turn these into nice Markdown links?

`[title](url)`

Reply:
Sure, why not.